winvnc/vnchooks/VNCHooks.cpp

```diff
@@ -463,18 +465,18 @@ inline void SendUpdateRect(SHORT x, SHORT y, SHORT x2, SHORT y2)
 	vwParam = MAKELONG(x, y);
 	vlParam = MAKELONG(x2, y2);
 
-	// Send the update to WinVNC
+	// Send the update to UltraVNC Server
 	PostThreadMessage(
 		 g_shdata->vnc_thread_id,
 		 g_shdata->UpdateRectMessage,
 		vwParam,
 		vlParam
 		);
-	HWND hWndRemote=FindWindow("WinVNC desktop sink", "WinVNC");
+	HWND hWndRemote=FindWindow("UltraVNC Server desktop sink", "WinVNC");

```

Comment:
Don't change
This name is used by other softwares.
You changed it to the same in vnchook and winvnc ..correct,
But changed it require a version check between both, older hooks will fail with new server and new hook wil fail on old server. The risk for possible mistakes it to high